README.md

```diff
@@ -187,6 +187,7 @@ Spatial-level transforms will simultaneously change both an input image as well
 | [RandomSizedCrop](https://albumentations.ai/docs/api_reference/augmentations/crops/transforms/#albumentations.augmentations.crops.transforms.RandomSizedCrop)                   | ✓     | ✓     | ✓      | ✓         |
 | [Resize](https://albumentations.ai/docs/api_reference/augmentations/geometric/resize/#albumentations.augmentations.geometric.resize.Resize)                                     | ✓     | ✓     | ✓      | ✓         |
 | [Rotate](https://albumentations.ai/docs/api_reference/augmentations/geometric/rotate/#albumentations.augmentations.geometric.rotate.Rotate)                                     | ✓     | ✓     | ✓      | ✓         |
+| [SafeRotate]()                                     | ✓     | ✓     | ✓      | ✓         |
```

Comment:
Please use this script to check and generate this table 
https://github.com/albumentations-team/albumentations/blob/master/tools/make_transforms_docs.py